src/main/java/frc/robot/subsystems/swerve/Swerve.java

```diff
@@ -299,37 +327,32 @@ public boolean isAtPosition(Pose2d pose2d) {
         return isAtXAxisPosition(pose2d.getX()) && isAtYAxisPosition(pose2d.getY()) && isAtAngle(pose2d.getRotation());
     }
 
+    private boolean isAtTranslationPosition(
+            double currentTranslationPosition, double targetTranslationPosition, double currentTranslationVelocity
+    ) {
+        return Math.abs(currentTranslationPosition - targetTranslationPosition) < SwerveConstants.TRANSLATION_TOLERANCE_METERS && Math.abs(
+                currentTranslationVelocity) < SwerveConstants.TRANSLATION_VELOCITY_TOLERANCE;
+    }
+    
```

Comment:
why does this get a velocity if it's `isAtTranslationPosition`?
consider changing the name